testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1563,3 +1563,16 @@
   raise 'File injection failed' unless return_code.zero?
   node.run('salt-call --local --file-root=/usr/share/susemanager/salt --module-dirs=/usr/share/susemanager/salt/ --log-level=info --retcode-passthrough state.apply ' + state)
 end
+
+When(/^I copy autoinstall mocked files on server$/) do
+  target_dirs = "/autoinstall/Fedora_12_i386/images/pxeboot/,/autoinstall/SLES15-SP2-x86_64/DVD1/boot/x86_64/loader/,/autoinstall/mock"
```

Comment:
```suggestion
  target_dirs = "/autoinstall/Fedora_12_i386/images/pxeboot/ /autoinstall/SLES15-SP2-x86_64/DVD1/boot/x86_64/loader/ /autoinstall/mock"
```

I have the feeling that you need to use a " " instead if "," to separate those dirs.
```
oscar@obarrios:/tmp> mkdir -p a/b,c/d,e/f
oscar@obarrios:/tmp> tree
.
├── a
│   └── b,c
│       └── d,e
│           └── f
```